### PR TITLE
moreutils: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by vitorhcl** [comment]
Well, the right spelling is "moreutils", as we can see in the website, should we enclose it wirh backticks or not?

File: pages/common/moreutils.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# moreutils
+
+> A collection of the UNIX tools that nobody thought to write long ago when UNIX was young.
```
Comment (tricantivu) [inline pages/common/moreutils.md:3]:
```suggestion
> A collection of UNIX tools.
```

**Comment by tricantivu** [comment]
> Well, the right spelling is "moreutils", as we can see in the website, should we enclose it wirh backticks or not?

I would say yes, however, other project names would have to be enclosed with backticks (e.g. `\`tldr\``)

**Comment by gutjuri** [comment]
Have we previously decided whether or not tldr will admit "index pages" that are not on a specific command?

**Comment by vitorhcl** [comment]
> Have we previously decided whether or not tldr will admit "index pages" that are not on a specific command?

Yes, see [#11844 (comment)](https://github.com/tldr-pages/tldr/pull/11844#issuecomment-1870083728) and [#5190 (comment)](https://github.com/tldr-pages/tldr/issues/5190#issuecomment-1852054873)

**Review by gutjuri — APPROVED** [review]

File: pages/common/moreutils.md (hunk 1 above)
Comment (gutjuri) [reply pages/common/moreutils.md:3]: This change is necessary, otherwise the text reads like an advertisement.

File: pages/common/moreutils.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# moreutils
+
+> A collection of UNIX tools.
+> moreutils isn't a command, but a set of commands.
```
Comment (kbdharun) [inline pages/common/moreutils.md:4]:
```suggestion
> Note: moreutils isn't a command, but a set of commands.
```
IG we can work around tldr-bot with a note instead.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**